break_glass/request_modal.json.txt

```diff
@@ -0,0 +1,47 @@
+{
+  "title": {
+    "type": "plain_text",
+    "text": "My App",
+    "emoji": true
+  },
+  "submit": {
+    "type": "plain_text",
+    "text": "Submit",
+    "emoji": true
+  },
+  "type": "modal",
+  "close": {
+    "type": "plain_text",
+    "text": "Cancel",
+    "emoji": true
+  },
+  "blocks": [
+    {
+      "type": "input",
+      "block_id": "block_issue_key",
+      "element": {
+        "type": "plain_text_input",
+        "action_id": "issue_key"
+      },
+      "label": {
+        "type": "plain_text",
+        "text": "Ticket Reference ID",
```

Comment:
Nit: "Reference Ticket ID" instead of "Ticket Reference ID".

The ticket is the reference proving that requester has a valid need for access, not the ID of the ticket.

Reply:
Done